shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -201,12 +201,7 @@ abstract class JSBackend(allowUnresolvedSymbols: Bool) {
     case NoCases => if (isQuoted) L(createASTCall("NoCases", Nil)) else R(NoCases)
   }
 
-  private def toJSOperator(op: Str) = op match {
-    case "+." => "+"
-    case "-." => "-"
-    case "*." => "*"
-    case _ => op
-  }
+  private def toJSOperator(op: Str) = op.takeWhile(_ =/= '.')
```

Comment:
I don't like this. The original was both clearer and had less potential for surprises and bugs.